test/test-style.js

```diff
@@ -128,12 +127,16 @@ function jsonDiff(actual, expected) {
 function testStyle(filename) {
   let hasErrors = false;
   let actual = fs.readFileSync(filename, 'utf-8').trim();
-  let expected = JSON.stringify(JSON.parse(actual), null, 2);
+  /** @type {import('../types').CompatData} */
+  let dataObject = JSON.parse(actual);
+  let expected = JSON.stringify(dataObject, null, 2);
+  let expectedSorting = JSON.stringify(dataObject, orderSupportBlock, 2);
 
-  const {platform} = require("os");
```

Comment:
Previously, `os` would be re‑imported for every single file.

This PR ensures that it’s only imported once.